packages/frontend/e2e/fixtures/test-setup.ts

```
@@ -0,0 +1,15 @@
+import { test as base, expect } from '@playwright/test';
+
+/**
+ * Custom test fixtures for ROSIE Middleware E2E tests.
+ * Provides authenticated context and API helpers.
+ */
+export const test = base.extend<{
+  apiBaseUrl: string;
+}>({
+  apiBaseUrl: async ({}, use) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)